akka-projection-r2dbc/src/main/scala/akka/projection/r2dbc/internal/R2dbcProjectionImpl.scala

```diff
@@ -362,6 +363,17 @@ private[projection] object R2dbcProjectionImpl {
                 Some(loadedEnvelope)
               }
             } else {
+              // FIXME do we need the same for all projection handler variants?
+              // FIXME not accepted may be that a producer filter decided to start including events,
+              //       how do we know for sure and what offset to start from?
+              env match {
+                case ee: EventEnvelope[_] if ee.sequenceNr > 0 =>
```

Comment:
Note that the first seqNr is always 1, not 0. Should this be `if ee.sequenceNr > 1`?